src/utils/Math.h

```diff
@@ -50,5 +51,31 @@ inline void LinearRegression(const std::vector<T1>& x, const std::vector<T1>& y,
 		beta2 = beta1 = 0.;
 }
 
+/**
+ * @brief Check whether two floats are within some maximum relative difference.
+ * Used to check for equality while taking floating-point errors into account.
+ * 
+ * Taken from AutoPas - src/autopas/utils/Math.h
+ * 
+ * @param a 
+ * @param b 
+ * @param maxRelativeDifference optional, default value 1e-9
+ * @return true 
+ * @return false 
+ */
+bool inline isNearRel(double a, double b, double maxRelativeDifference = 1e-9) 
+{
+	const auto greaterNumber = std::max(std::abs(a), std::abs(b));
+	const auto absoluteDifference = maxRelativeDifference * greaterNumber;
+	const auto diff = std::abs(a - b);
+	return diff <= absoluteDifference;
```

Comment:
Use brackets to make return value more obvious
```suggestion
	return (diff <= absoluteDifference);
```